Pipfile

```diff
@@ -10,16 +10,14 @@ click = "~=8.1"
 decorator = "~=5.1"
 isort = "~=5.10.0"
 mypy = "~=0.950"
-panther-analysis-tool = "~=0.24.0"
```

Comment:
should this be replaced with the higher version?

Reply:
`panther-analysis-tool` was listed in two sections, *packages* and *dev-packages*. I trimmed it down to just the one *packages* entry.